wandb/cli/cli.py

```diff
@@ -856,8 +869,58 @@ def _parse_settings(settings):
         or api.settings("project")
         or util.auto_project_name(config.get("program"))
     )
+
+    _launch_scheduler_spec = None
+    if queue is not None:
+        wandb.termlog("Using launch 🚀 queue: %s" % queue)
+
+        # Because the launch job spec below is the Scheduler, it
+        # will need to know the name of the sweep, which it wont
+        # know until it is created,so we use this placeholder
+        # and replace inside UpsertSweep in the backend (mutation.go)
+        _sweep_id_placeholder = "WANDB_SWEEP_ID"
+
+        # Launch job spec for the Scheduler
+        # TODO: Keep up to date with Launch Job Spec
+        _launch_scheduler_spec = json.dumps(
+            {
+                "queue": queue,
+                "run_spec": json.dumps(
+                    construct_launch_spec(
+                        os.getcwd(),  # uri,
+                        api,
+                        f"Scheduler.{_sweep_id_placeholder}",  # name,
+                        project,
+                        entity,
+                        None,  # docker_image,
+                        "local-process",  # resource,
+                        [
+                            "wandb",
```

Comment:
hmm, I see that you hijacked the `wandb launch-agent` comment with a `--queues` arg which essentially starts the sweep scheduler. Similar to my comment below, I would prefer moving this to another sub command in the SDK, probably the `wandb sweep-agent` would be my preference so we can keep `wandb launch-agent` clean